tests/test_requests.py

```diff
@@ -460,9 +460,14 @@ def test_HTTP_200_OK_PUT(self, httpbin):
         r = requests.put(httpbin('put'))
         assert r.status_code == 200
 
-    def test_BASICAUTH_TUPLE_HTTP_200_OK_GET(self, httpbin):
-        auth = ('user', 'pass')
-        url = httpbin('basic-auth', 'user', 'pass')
+    @pytest.mark.parametrize(
+        'username, password', (
+            ('user', 'pass'),
+            (u'имя'.encode('utf-8'), u'пароль'.encode('utf-8')),
+        ))
+    def test_BASICAUTH_TUPLE_HTTP_200_OK_GET(self, httpbin, username, password):
+        auth = (username, password)
+        url = httpbin('basic-auth', username, password)
```

Comment:
Ah, sorry, this test fails because of the way the URL has been constructed here. The httpbin url fixture doesn't support bytes here.

Instead of doing the test this way, let's remove httpbin entirely. We should be able to check what the correct value is by using the prepared request flow:

```python
r = Request('GET', httpbin('get'), auth=(username, password))`
p = r.prepare()

assert p['Authorization'] == "whatever the auth string should be"
```

Want to change the test to that form?